app/templates/partials/introduction/preview.html

```diff
@@ -0,0 +1,44 @@
+<h2 class="saturn">{{ intro.title }}</h2>
+{% for content in intro.content %}
+<h3 class="venus u-mt-s">{{ content.title }}</h3>
+<div class="qa-intro-description">
+    {{ content.description }}
+</div>
+{% endfor %}
+<div class="accordion js-accordion">
+        <div class="mars accordion__controls">
+            <button class="accordion__control js-accordion-open-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" tabindex="0" aria-hidden="true">Show all</button>
+            <button class="accordion__control js-accordion-close-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Hide all" tabindex="0" aria-hidden="true">Hide all</button>
+        </div>
+        <dl class="js-accordion-content">
+            {% for question in intro.questions %}
+            {% set outer_loop = loop %}
+            <dt class="accordion__title venus js-accordion-title" data-js-accordion-event-label="{{question.question}}">
```

Comment:
Cool I'll have a read of that. I've heavily cribbed that from http://a11yproject.com/patterns.html (under accordions & tabs), will be interesting to compare.